symphony/bdk/core/config/bdk_config_parser.py

```diff
@@ -0,0 +1,25 @@
+import yaml
+import json
+from symphony.bdk.core.config.exception.bdk_config_exception import BdkConfigException
+
+
+class BdkConfigParser:
+
+    @classmethod
+    def parse(cls, config_content: str):
+        config_data = cls.parse_input(config_content)
+        return config_data
+
+    @staticmethod
+    def parse_input(config_content: str):
+        try:
+            return json.loads(config_content)
+        except json.JSONDecodeError:
+            pass
+
+        try:
+            return yaml.safe_load(config_content)
+        except yaml.YAMLError as e:
+            pass
+
+        raise BdkConfigException("Config file is not the right file type")
```

Comment:
Please be more specific with something like : `Config file format is neither JSON nor YAML"

Reply:
I will change it to `Config file has a wrong format.` as it can be a misformatted YAML file. 
There is a test using an invalid yaml file btw.

Reply:
We can add that information in the logs. After implementing that.